test/cmdlineTests.sh

```diff
@@ -302,25 +320,25 @@ test_solc_behaviour "${0}" "ctx:=/some/remapping/target" "" "" 1 "" "Invalid rem
 printTask "Running general commandline tests..."
 (
     cd "$REPO_ROOT"/test/cmdlineTests/
-    for tdir in ${*:-*/}
+    (( ${#selected_tests[@]} > 0 )) || selected_tests=(*)
```

Comment:
> What's the purpose of this? If there are no selected tests, select all tests?
Shouldn't it be (( ${#selected_tests[@]} == 0 )) && selected_tests=(*) or (( ${#selected_tests[@]} > 0 )) || selected_tests=(*) for that?

sorry for that, it is fixed now.